react-client/src/components/SharedContent/SharedContentSettings.tsx

```diff
@@ -134,6 +134,12 @@ export default function SharedContentSettings(
         return i18n.get['Folder'];
       case 'VirtualFolder':
         return i18n.get['VirtualFolders'];
+      case 'iTunes':
+        return i18n.get['ItunesLibrary'];
+      case 'iPhoto':
+        return i18n.get['IphotoLibrary'];
+      case 'Aperture':
+        return i18n.get['ApertureLibrary'];
```

Comment:
I would be surprised if these features work, I don't think they even exist in macOS anymore. That's why I didn't include them in the web settings